fontbox/src/main/java/org/apache/fontbox/ttf/CmapTable.java

```diff
@@ -81,9 +81,10 @@ void read(TrueTypeFont ttf, TTFDataStream data) throws IOException
             cmap.initData(data);
             cmaps[i] = cmap;
         }
+        int numberOfGlyphs = ttf.getNumberOfGlyphs();
```

Comment:
I think the JVM is able to optimize this by using `hoisting` (see e.g.: https://advancedweb.hu/jvm-jit-optimization-techniques-part-2/). Imho there is no need to do it by hand and increases the lines of code.

Reply:
it could be